src/lib/import-export/import/validators/sql-validator.ts

```diff
@@ -0,0 +1,30 @@
+import path from 'path';
+import { BackupContents } from '../types';
+import { Validator } from './Validator';
+
+export class SqlValidator implements Validator {
+	canHandle( fileList: string[] ): boolean {
+		return fileList.length === 1 && fileList[ 0 ].endsWith( '.sql' );
```

Comment:
Based on the PR's testing instructions, I'm not sure if this part is ready and should be tested. In any case, I'm out of curiosity why we only allow importing a single SQL file. I presume it's related to the expectation of having a single big file with all SQL statements, but I wonder if we should also support a zipped file that contains multiple SQL files. WDYT?

This comment is tangentially related to https://github.com/Automattic/dotcom-forge/issues/8282, as it mentions support `.zip` and `.tar.gz` for single `.sql` files.

Reply:
> but I wonder if we should also support a zipped file that contains multiple SQL files. WDYT?

As this is already a big one, after discussion with Wojtek, I opened a new ticket: https://github.com/Automattic/dotcom-forge/issues/8282

Reply:
Sound good, let's move the discussion to that ticket.